library/build.gradle.kts

```diff
@@ -20,11 +20,17 @@ kotlin {
     iosSimulatorArm64()
     linuxX64()
 
-    sourceSets.commonMain.get().dependencies {
-        //put your multiplatform dependencies here
-    }
-    sourceSets.commonTest.get().dependencies {
-        implementation(libs.kotlin.test)
+    sourceSets {
```

Comment:
Here you changed the way which Sebastian recommended

Reply:
it is not good idea because it works only for common source sets. for any other user will have to use "by getting" syntax

Reply:
another point is I'd prefer to have a common approach in our templates.
we don't use `sourceSets.` for each source set

Reply:
With 1.9.20 there will be static accessors not just for commonMain but other default source sets (including nativeMain, iosMain and etc)